sensor/kubernetes/listener/watcher/complianceoperator/availabilitychecker_test.go

```diff
@@ -0,0 +1,74 @@
+package complianceoperator
+
+import (
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAllCOResourcesAreAddedToTheAvailabilityChecker(t *testing.T) {
+	ac := NewComplianceOperatorAvailabilityChecker()
+	pwd, err := os.Getwd()
+	require.NoError(t, err)
+
+	fset := token.NewFileSet()
+	file, err := parser.ParseFile(fset, path.Join(pwd, "../../../../../pkg/complianceoperator/api.go"), nil, 0)
+	require.NoError(t, err)
+
+	whileList := []string{
+		"groupVersion",
+		"requiredAPIResources",
+	}
+
+	resFinder := &resourcesFinder{}
+	ast.Walk(resFinder, file)
+
+	var notFound []string
+finderLoop:
+	for _, resource := range resFinder.resources {
```

Comment:
Add an `require.NotEmpty(t, resFinder.resources)` to assert that the resources were parsed. (Empty result would make this test pass)

Reply:
Done 👍